src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -74,6 +74,8 @@
   "minecolonies.config.builderinfiniteresources.comment": "Should the Builder and Miner build without resources? (This also turns off what they produce.) [Default: false]",
   "minecolonies.config.limittoonewarehousepercolony": "Warehouse Limit",
   "minecolonies.config.limittoonewarehousepercolony.comment": "Should there be a max of 1 warehouse per colony?. [Default: true]",
+  "minecolonies.config.limittoonetavernpercolony": "Tavern Limit",
+  "minecolonies.config.limittoonetavernpercolony.comment": "Should there be a max of 1 Tavern per colony?. [Default: true]",
```

Comment:
```suggestion
  "minecolonies.config.limittoonetavernpercolony.comment": "Should there be a max of 1 Tavern per colony? [Default: true]",
```

Reply:
I didn't read it, i just copy pasted it xD

Reply:
My fault for not seeing the period in my language PR 😄